internal/grpc/interceptors/eventsmiddleware/conversion.go

```diff
@@ -201,14 +203,39 @@ func OCMCoreShareCreated(r *ocmcore.CreateOCMCoreShareResponse, req *ocmcore.Cre
 		}
 	}
 	return events.OCMCoreShareCreated{
-		ShareID:       r.GetId(),
-		Executant:     executant.GetId(),
-		Sharer:        req.GetSender(),
-		GranteeUserID: req.GetShareWith(),
-		ItemID:        req.GetResourceId(),
-		ResourceName:  req.GetName(),
-		CTime:         r.GetCreated(),
-		Permissions:   permissions,
+		ShareID:      r.GetId(),
+		Executant:    executant.GetId(),
+		Sharer:       req.GetSender(),
+		Grantee:      req.GetShareWith(),
+		ItemID:       req.GetResourceId(),
+		ResourceName: req.GetName(),
+		CTime:        r.GetCreated(),
+		Permissions:  permissions,
+	}
+}
+
+func OCMCoreShareDelete(r *ocmcore.DeleteOCMCoreShareResponse, req *ocmcore.DeleteOCMCoreShareRequest, executant *user.User) events.OCMCoreShareDelete {
+	timestampStrInSec := utils.ReadPlainFromOpaque(r.GetOpaque(), "timestamp")
+	timestampIntInSec, err := strconv.ParseUint(timestampStrInSec, 10, 64)
```

Comment:
```suggestion
	timestampIntInSec, err := strconv.ParseUint(utils.ReadPlainFromOpaque(r.GetOpaque(), "timestamp"), 10, 64)
```
Don't like it like that? Line too long?

Reply:
Ok for both. It felt easier to read